website/pages/en/resources/roles/delegating.mdx

```diff
@@ -40,25 +62,29 @@ As a Delegator, it's important to understand the following:
 
 - In order to be safe, you should calculate your potential return when delegating to an Indexer. For example, you might calculate how many days it will take before you have earned back the 0.5% tax on your delegation.
 
-### The delegation unbonding period
+### The Delegation Unbonding Period
+
+When a Delegator chooses to undelegate, their tokens are subject to a 28-day unbonding period.
+
+This means they cannot transfer their tokens or earn any rewards for 28 days.
 
-Whenever a Delegator wants to undelegate, their tokens are subject to a 28-day unbonding period. This means they cannot transfer their tokens or earn any rewards for 28 days.
+After the unbonding period, GRT will return to your crypto wallet.
```

Comment:
I just noticed that the UI calls this the "undelegation period". cc @IainM22, maybe that was intentional because it's more user-friendly? Just wanted to raise it.

Reply:
Thank you for flagging! Yes, undelegation works best here.